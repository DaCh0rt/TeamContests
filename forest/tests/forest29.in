519 616
7964 2185
-3782 -9293
5764 7765
5576 1716
-8631 2956
4371 6862
9495 1724
-4175 -5911
8604 -8144
-9104 8802
1490 1034
4639 -4874
-2488 -7459
-5232 331
-6733 4184
1305 7332
-1884 2249
-3550 3065
4413 -8028
-682 8447
6119 -6940
-957 -5967
8856 7197
9806 -1908
-2961 -8906
-7215 -4323
-53 6446
-4238 -1602
3159 3271
9315 1674
-2515 -5743
4039 4225
5017 -4302
-8502 -9924
-3411 -3084
5019 -2569
-6517 7856
8571 -5403
-2742 6102
1844 -5779
-4931 9518
878 2796
4689 -4553
8945 -7465
-1687 -6862
3327 -849
-5980 -5131
-2786 -3398
7195 -8677
-6341 -6712
-3925 7775
5698 -7270
-4098 6109
-6475 -3757
-7105 9047
7853 9136
6440 -4252
7579 7261
-949 8072
-1190 -5448
1089 -8474
-2482 3697
8501 -4455
8274 5166
3722 -8695
-1015 -1939
6349 -6470
4587 -2949
-5714 2885
-4586 3598
-4814 8696
-7471 512
7219 -3306
-2593 -6637
8148 -2306
5453 -9853
9992 -2701
9181 6600
-778 9072
4915 -7826
-533 9956
-9816 5011
-7540 7554
5911 -5977
-1006 8462
-9059 -2001
5087 8889
-4925 -2550
-3889 -966
-662 -4065
-2468 9618
-7275 -4924
-9162 6501
9670 1317
8067 -9586
9249 3781
-4088 3554
-3312 4092
2443 -8949
9606 -6337
-8458 -232
3803 3759
-8969 7563
-6330 -9167
8097 3983
1637 -1141
-7352 5392
7870 -360
9585 9895
-2641 -1341
9229 -2244
-8301 -1054
-4462 -1625
4609 3129
2006 -8028
-7976 -5925
8787 2059
-3762 -424
5288 9
-1431 -5787
-4703 -6423
-9986 -2743
917 2029
-5660 318
3336 4188
2245 -66
8029 -3787
-8277 8545
3550 8709
-3420 -7522
-2656 -562
9796 -4780
4623 3522
7701 9325
8483 3654
-6258 -5732
-1642 4642
4330 1278
3870 9145
2938 4843
-5695 -2850
-8285 5667
-3510 5197
7509 -4802
6626 6107
9801 -2431
-2615 -9076
-2725 6775
1044 -4264
-6921 -2572
-9719 -3101
1996 -463
-6477 5836
-5738 -8879
6467 7792
-4944 -2987
-7498 -7981
159 854
2821 8951
-873 443
8148 -1048
7791 744
6646 8080
-1874 2918
4280 -7116
-9364 4608
7432 8362
6458 9716
-3304 676
6259 -6458
187 6632
4944 9571
4772 -546
8258 -153
-9070 -7010
-408 1593
-1240 3917
3906 2360
-4920 -8933
-2738 -7083
162 -81
-8009 -9925
-4252 -6236
-3833 9796
6565 -9835
1686 -8690
8326 6001
778 1522
344 9416
2201 6793
3497 9541
-3008 3861
6609 -1593
1252 8971
7386 -3877
388 -2863
7507 6882
-535 4405
-2759 5129
383 4614
-9936 -6422
-8946 5639
3823 -1194
6490 -1379
-4593 -6879
-7640 -8787
4603 9909
-6826 7307
3119 3859
-1446 -2561
-1417 -2772
7098 4868
1006 -8620
8189 -2907
-8269 3235
-2689 -5151
377 -4753
-429 -9765
7895 -5647
-9675 -2039
8350 -1827
1236 -6746
-5433 -2170
2465 978
4283 3325
429 -5943
8474 -9604
-4720 -9319
5334 -2073
-6407 -3575
8110 4180
-5633 5472
7964 -4033
-4601 -418
2934 -4569
-8443 6777
-5249 1856
4566 6240
7495 1588
-6243 -430
-8882 -8833
5584 -2572
6217 -5999
-7413 4243
-393 -6342
3589 -3356
5009 -8469
3446 8967
-9843 -1555
-2362 -9307
-5122 2331
-8987 -2621
-8252 -4096
-7012 -1630
3498 6948
1009 6637
587 2863
2085 -4143
-7404 -5150
5557 8838
2667 1951
4488 -444
-6568 -5551
5070 -8705
6491 9316
3679 -2759
568 8393
-5957 -4530
5105 624
4382 9152
-6256 -8854
5459 -9436
9986 3777
-1676 -8673
-5025 4386
-380 5087
-1572 374
-6139 -5361
3713 -8239
-4803 -8299
8473 -7766
731 -7743
3833 8585
-3504 -9756
1602 -9945
3089 -9564
6490 3784
-2729 2974
5262 5843
-7091 7129
-5817 -3297
-8468 8885
-4401 -5431
-1889 -7217
-5617 -4134
8709 7096
-573 -4784
1746 7202
2196 2354
3770 8980
8591 -9146
2973 -520
-6040 5898
6902 -146
9113 -833
1487 -7552
8949 -1467
8255 979
-5107 9334
5692 7286
-5067 7096
-5026 -5276
-7218 9271
-4559 2381
9428 8137
9248 796
-5963 -9623
593 -3068
-9517 -5752
3595 9633
-8466 8686
-7333 6537
-4483 -8881
-1221 3843
762 5430
9606 -599
811 -2842
-3141 7764
6975 -747
-2738 -6128
-9850 -3716
-3193 5835
5628 -1796
9610 -258
-1038 -9354
-2712 5069
9254 -8287
3630 6585
343 416
-466 -5995
2280 8570
1174 6482
-1673 -9813
-3996 -8729
-5767 -7979
-2428 -3059
-1377 -6348
-9197 -226
-8821 -36
6633 -3783
-3590 -3157
-4142 4053
9701 -4723
2943 -6780
1398 -9623
4055 8096
-5217 -6172
-5639 2166
5663 8471
-6206 -525
-1136 -5382
1545 5506
3641 -7942
-5591 8176
-1867 -7581
5030 -2387
8190 7856
-1832 -2512
-1674 8320
7967 -3209
4745 8289
-3496 4135
1634 2397
346 -3582
-1462 9422
3518 -2402
1401 3675
3152 -2476
6998 4993
-4433 2495
-2125 2819
9233 1015
2983 -1290
7118 6975
3894 -6393
-2133 -6157
-80 -3815
8867 -3442
-2646 6543
5414 -9213
6946 -7235
7845 -224
-6919 8864
3295 9664
8389 8292
1328 -8838
3126 8796
-2494 6491
3994 -8402
-9518 8567
-7613 9271
3994 1831
6052 -6914
-5213 -9541
-6050 221
1037 -3138
-2208 -6074
-6 -8344
8109 9641
-1596 -4691
-7993 8556
4920 5045
-75 3027
9166 4929
-3455 -1918
2893 2235
-4455 -1144
-5031 9208
7338 -6896
5919 5989
4081 -7388
-4149 6122
5224 2933
6712 -880
-3121 8654
8969 382
-6251 7406
3854 -5109
-1833 2679
-5370 6139
2009 7697
-3780 949
4219 -4182
3288 -8098
9274 -8143
1200 8848
-9394 -4305
273 -9041
-8932 -9557
5534 8234
9567 8544
-7308 -1559
-5413 50
-9172 -251
-2917 6483
-1100 -5920
9156 -5603
1654 2086
-2344 1552
3292 -6616
-1868 8800
-6520 -9946
-6840 -178
-2379 4786
-5918 6538
-7048 -505
-2344 -3544
-4145 -6901
-3096 -4952
1399 -6402
2279 9023
247 -4798
6481 -8211
-4147 -1519
9953 -612
7098 253
2203 5924
-6196 -6025
3881 5216
8299 1231
-6132 -6128
1984 8143
-1769 -1714
-3149 -3977
5980 -8708
-7319 -3184
4416 -4799
4508 -5323
-1818 -3414
-2515 -4051
-6515 9169
-225 -5354
2945 -5313
-3507 3659
2905 -8077
-4542 5362
-8529 9887
-221 -3947
-693 -1340
-4880 9786
-19 -7806
5163 3010
8500 -6142
-410 2594
221 -3583
-9984 6839
9725 -5821
1649 1246
3389 4860
-1081 8027
6887 -4141
1854 9596
6125 4299
-7378 -9608
6499 -2868
-227 -7042
-463 390
-9980 -7072
1095 5729
-7995 1489
3814 -6958
-3501 -4519
-8467 6054
-3582 -153
-9107 -6284
-6460 6992
-6843 3837
-3890 1479
-6537 -533
