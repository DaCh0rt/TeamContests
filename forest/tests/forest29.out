79370.36 435398411.56
